Makefile

```
@@ -6,7 +6,7 @@ PYTEST ?= $(shell command -v pytest 2>/dev/null || echo .venv/bin/pytest)
 RUFF   ?= $(shell command -v ruff 2>/dev/null || echo .venv/bin/ruff)
 MYPY   ?= $(shell command -v mypy 2>/dev/null || echo .venv/bin/mypy)
 
-.PHONY: install test test-unit test-integration test-cov lint format typecheck check ci clean docs-serve docs-build
+.PHONY: install test test-unit test-integration test-cov lint format typecheck check ci clean docs-serve docs-build dashboard
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)